### DISCUSSION TITLE
How do I use three.js to draw such a model?

### DISCUSSION THREAD

**Discussion by wenze159632** [materialsproject/pymatgen#3769] [Q&A] [answered]
I want to add such a 3D crystal model to my own portfolio website. Please tell me how to create this effect.

![1713448776832](https://github.com/materialsproject/pymatgen/assets/153369683/ba75a3d9-d731-4386-bac9-3186f74a6e18)

**Comment by shyuep** [comment, marked as answer]
This has nothing to do with pymatgen. Please send your questions to materials project.

**Reply by wenze159632** [reply]
okay, thanks for your answer
